documentation/source/regulator.rst

```diff
@@ -0,0 +1,111 @@
+##################################
+The cocotb ``regulator`` Testbench
+##################################
```

Comment:
Just like in the rescap example, I think it might be a good idea to move the note about only working with Cadence simulators up. And add a note about the standard tests depending upon matplotlib.